package.json

```diff
@@ -52,6 +53,7 @@
     },
     "devDependencies": {
         "husky": "^5.0.9",
-        "prettier": "^2.2.1"
+        "prettier": "^2.2.1",
+        "zipcodes-perogi": "^1.4.201908"
```

Comment:
this shouldn't be in both regular dependencies AND devdependencies

Reply:
After actually looking at your code I think we can just leave it in devDeps, right?

Reply:
It should not have been in `dependencies`! I don't know how that happened!
Done in 008cbd79f636bdac5f61cce0283251b5c12e7930